manifests/o/ovito/ovito-basic/3.0.1/ovito.ovito-basic.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ovito.ovito-basic
+PackageVersion: 3.0.1
+PackageLocale: en-US
+Publisher: ovito

```

```suggestion
Publisher: OVITO GmbH, Germany
```
This must match Control Panel.